src/cool_seq_tool/mappers/mane_transcript.py

```diff
@@ -524,9 +524,7 @@ def _validate_index(
         end_pos = pos[1] + coding_start_site
         if self.seqrepo_access.get_reference_sequence(
             ac, start_pos, end_pos, residue_mode=ResidueMode.INTER_RESIDUE
-        )[
-            0
-        ]:  # noqa E501
+        )[0]:  # noqa E501
```

Comment:
I think we can remove the noqa
```suggestion
        )[0]:
```